modules/email/src/services/email.service.ts

```diff
@@ -128,6 +142,55 @@ export class EmailService {
     if (params.attachments) {
       builder.addAttachments(params.attachments as Attachment[]);
     }
-    return this.emailer.sendEmail(builder);
+
+    const sentMessageInfo = await this.emailer.sendEmail(builder);
```

Comment:
I would advise to disable creating a new storage entry if the email is a resend. That can work by either creating a database record that points to the existing email content, or by updating the existing record